internal/confmapprovider/discovery/bundle/bundle.d/receivers/smartagent-postgresql.discovery.yaml

```diff
@@ -52,17 +61,12 @@ smartagent/postgresql:
           first_only: true
           log_record:
             severity_text: info
+            append_pattern: true
             body: >-
               Make sure your PostgreSQL database has
               `shared_preload_libraries = 'pg_stat_statements'`
               in the postgresql.conf file and that
               `CREATE EXTENSION IF NOT EXISTS pg_stat_statements;`
               has been run for each database you would like to monitor.
-        - regexp: 'could not monitor postgresql server: failed to determine total_time column name'
```

Comment:
This prefix occurs more frequently than I thought, including w/ the above unreachable network issue (definitely not warranting a partial status).